docs/sources/developers/plugins/build-a-logs-data-source-plugin.md

```diff
@@ -93,11 +102,11 @@ frame.add({ time: 1589189406480, content: 'user logged in' });
 
 ## Extract detected fields from your logs
 
-You can add additional information about each log line by adding more data frame fields.
+Add additional information about each log line by supplying more data frame fields.
 
-If a data frame has more than one text field, then Grafana assumes the first field in the data frame to be the actual log line. Any subsequent text fields are treated as [detected fields]({{< relref "../../explore/#labels-and-detected-fields" >}}).
+If a data frame has more than one text field, then Grafana assumes the first field in the data frame to be the actual log line. Grafana treats subsequent text fields as [detected fields]({{< relref "../../explore/#labels-and-detected-fields" >}}).
 
-While you can add any number of custom fields to your data frame, Grafana comes with a couple of dedicated fields: `levels` and `id`. Let's have a closer look at each one.
+Although you can add any number of custom fields to your data frame, Grafana comes with a couple of dedicated fields: `levels` and `id`. Let's have a closer look at each one.
```

Comment:
Suggestion: "Any number of custom fields can be added to your data frame; Grafana comes with two dedicated fields: levels and id."

Reply:
Also suggest we extend the "Unique log lines" subheader to indicate that it is id, otherwise you lose the connection between saying which two you're going to explain.

Reply:
Issue is fixed in latest commit.